core/commands/dag/dag.go

```diff
@@ -13,9 +13,9 @@ import (
 	cid "github.com/ipfs/go-cid"
 	cidenc "github.com/ipfs/go-cidutil/cidenc"
 	cmds "github.com/ipfs/go-ipfs-cmds"
-	//gipfree "github.com/ipld/go-ipld-prime/impl/free"
-	//gipselector "github.com/ipld/go-ipld-prime/traversal/selector"
-	//gipselectorbuilder "github.com/ipld/go-ipld-prime/traversal/selector/builder"
+	// gipfree "github.com/ipld/go-ipld-prime/impl/free"
+	// gipselector "github.com/ipld/go-ipld-prime/traversal/selector"
+	// gipselectorbuilder "github.com/ipld/go-ipld-prime/traversal/selector/builder"
```

Comment:
This is on us, but WTF does commented out import in our code ?
Should be removed (in an other PR).

Reply:
https://github.com/ipfs/kubo/pull/10082